src/Security/CMSSecurity.php

```diff
@@ -163,28 +170,23 @@ public function GetLoginForms()
     public static function enabled()
     {
         // Disable shortcut
-        if (!static::config()->reauth_enabled) {
+        if (!static::config()->get('reauth_enabled')) {
             return false;
         }
 
-        // Count all cms-supported methods
-        $authenticators = Authenticator::get_authenticators();
-        foreach ($authenticators as $authenticator) {
+        /** @var [] $authenticators */
+        $authenticators = Security::config()->get('authenticators');
```

Comment:
Yep, definitely. It was a copy-paste from before passing in the Authenticator type required. Good spot!